public/assets/ts/admin/image-store.ts

```diff
@@ -1,209 +1,185 @@
-import { showToast } from "./toast";
```

Comment:
I don't know why it says I've changed this entire file. I removed the upload logic and put it into it's own file. That's all :(